media-gfx/blender/blender-2.82a.recipe

```diff
@@ -0,0 +1,321 @@
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libtbb$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libtinyxml$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libyaml_cpp$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	lib:libavdevice$secondaryArchSuffix # REQUIRED
+	"
+BUILD_PREREQUIRES="
+	numpy${secondaryArchSuffix}_python3
+	requests_python3
+	cmd:cmake
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:python${PYTHON_VERSION}
+	cmd:sed
+	cmd:strip
+	"
+
+defineDebugInfoPackage blender$secondaryArchSuffix \
+	$appsDir/Blender/Blender
+
+BUILD()
+{
+	PORT_SUBFOLDER=`echo $portVersion | sed 's/.$//'`
+
+	INCLUDE_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`${secondaryArchSubDir}"
+
+	# Haiku specific settings
+	BLENDER_PLATFORM_SPECIFIC="-DCMAKE_INSTALL_PREFIX:PATH=$appsDir/Blender \
+		-DPYTHON_VERSION=${PYTHON_VERSION} \
+		-DPYTHON_INCLUDE_DIR=`finddir B_SYSTEM_HEADERS_DIRECTORY`/python${PYTHON_VERSION}m/ \
+		-DPYTHON_LIBRARY=`finddir B_SYSTEM_LIB_DIRECTORY`${secondaryArchSubDir}/libpython${PYTHON_VERSION}m.so.1.0 \
+		-DPYTHON_EXECUTABLE=/bin/python${PYTHON_VERSION} \
+		-DPYTHON_NUMPY_PATH=`finddir B_SYSTEM_LIB_DIRECTORY`/python${PYTHON_VERSION}/vendor-packages \
+		-DPYTHON_REQUESTS_PATH=`finddir B_SYSTEM_LIB_DIRECTORY`/python${PYTHON_VERSION}/vendor-packages \
+		-DOPENCOLLADA_COLLADABASEUTILS_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADABaseUtils \
+		-DOPENCOLLADA_COLLADAFRAMEWORK_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADAFramework \
+		-DOPENCOLLADA_COLLADASAXFRAMEWORKLOADER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADASaxFrameworkLoader \
+		-DOPENCOLLADA_COLLADASTREAMWRITER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADAStreamWriter \
+		-DOPENCOLLADA_GENERATEDSAXPARSER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/GeneratedSaxParser \
+		-DOPENSUBDIV_INCLUDE_DIR=${INCLUDE_DIR} \
+		-DZLIB_LIBRARY_RELEASE=`finddir B_SYSTEM_LIB_DIRECTORY`${secondaryArchSubDir}/libz.so.1 \
+		-DWITH_GHOST_SDL=ON \
+		-DWITH_BUILDINFO=OFF \
+		-DWITH_PYTHON_INSTALL=YES \
+		-DWITH_PYTHON_INSTALL_NUMPY=YES \
+		-DWITH_PYTHON_INSTALL_REQUESTS=YES \
+		-Wno-dev"
+
+	# Release settings breaks build on Haiku
+	BLENDER_BUGOUS="-DWITH_CYCLES_LOGGING=OFF \
+		-DWITH_GAMEENGINE_DECKLINK=OFF \
+		-DWITH_GTESTS=OFF \
+		-DWITH_SYSTEM_EIGEN3=ON \
+		-DWITH_SYSTEM_LZO=ON"
+
+	# Missing deps on Haiku
+	BLENDER_MISSING="-DWITH_JACK=OFF \
+		-DWITH_MEM_JEMALLOC=OFF \
+		-DWITH_INPUT_NDOF=OFF"
+
+	# Release build settings
+	BLENDER_RELEASE="-DWITH_ALEMBIC=ON \
+		-DWITH_BULLET=ON \
+		-DWITH_CODEC_AVI=ON \
+		-DWITH_CODEC_FFMPEG=ON \
+		-DWITH_CODEC_SNDFILE=ON \
+		-DWITH_CYCLES=ON \
+		-DWITH_CYCLES_OPENSUBDIV=ON \
+		-DWITH_CYCLES_OSL=ON \
+		-DWITH_FFTW3=ON \
+		-DWITH_LIBMV=ON \
+		-DWITH_LIBMV_SCHUR_SPECIALIZATIONS=ON \
+		-DWITH_GAMEENGINE=ON \
+		-DWITH_COMPOSITOR=ON \
+		-DWITH_FREESTYLE=ON \
+		-DWITH_GHOST_XDND=ON \
```

Comment:
You'll want to disable this. I believe this flag communicates that we support X11 drag and drop functionality, which we don't. :)